frontend/src/lib/validation.ts

```diff
@@ -200,13 +203,13 @@ export function validateCloudsYaml(yamlValue: string, cloudValue: string, t: TFu
     return undefined
 }
 
-export function validateWebURL(url: string, t: TFunction, supportedProtocols?: string[]) {
-    const protocols = supportedProtocols ? supportedProtocols : ['http', 'https']
+export function validateWebURL(url: string, _item: unknown, t?: TFunction) {
+    t = t ? t : (value) => value
     if (
         validator.isURL(url, {
             require_protocol: true,
             require_valid_protocol: true,
-            protocols: protocols,
+            protocols: ['http', 'https'],
```

Comment:
Why did you change this? The credentials form specifically wants to allow only `https` but this hard-codes it to include `http`.

Reply:
i recall a merge conflict a while back I think i meant to revert then.  Reverted now.